README.md

```diff
@@ -171,6 +171,9 @@ If you cannot use Serilog-expressions due to framework compatibility - you can i
 | `exceptionHandler`       | `Action<Exception>`    | This function is called when an exception occurs when using `DatadogConfiguration.UseTCP=false` (the default configuration). |
 | `detectTCPDisconnection` | `bool`                 | Detect when the TCP connection is lost and recreate a new connection. |
 | `formatter`              | `ITextFormatter`       | A custom formatter implementation to change the format of the logs |
+| `maxMessageSize`         | `int`                  | The maximum size in bytes of a message before it is truncated |
```

Comment:
Let's clarify that the message is split into multiple chunks when it exceeds this configured size (I think "truncated" means that the message is truncated to the max size and that the rest of the message is dropped; whereas the behavior you implemented is to split it into multiple chunks)